tests/test_geoutils.py

```diff
@@ -0,0 +1,175 @@
+"""
+Test functions for geoutils (not using unittest)
+"""
+import os
+import geoutils.georaster as gr
+import geoutils.geovector as gv
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+fn_img = os.path.join('/home/atom/code/devel/libs/GeoUtils/tests','data','LE71400412000304SGS00_B4_crop.TIF')
```

Comment:
You should define the path relative to the source code!

Reply:
The best thing would be to define a path to the test data folder somewhere, like done in georaster: https://github.com/GeoUtils/georaster/blob/0df19bfd91fd10e92beb54308428e212e1d19c1a/georaster/georaster.py#L147
You could even define a dictionary with paths to several typical data set (the Landsat crops, the RGI outlines etc).
Then you can use this to run your tests, see https://github.com/GeoUtils/georaster/blob/master/test/test_georaster.py
You may also want to copy some of those tests already.

Reply:
The path relative to source code is defined just below, it was commented as it works only on command line, and I prefer executing from the console when I test :/. Maybe you know a solution that works both cases? (it's a pain to switch from live console in editor to command line just for testing...)
I think it's better to have the path directly defined in test_geoutils rather than "hidden" all the way to georaster.py (as in https://github.com/GeoUtils/georaster/blob/0df19bfd91fd10e92beb54308428e212e1d19c1a/georaster/georaster.py#L147)

From talking with Fabien (raster_tools slack), we concluded it was probably better to use pytest than unittest. So we'd have to rework the tests a bit before copying them (which is why I did not do it).

Reply:
Regarding finding the path to a code, I think this solves your issue: https://stackoverflow.com/questions/2632199/how-do-i-get-the-path-of-the-current-executed-file-in-python/5748583#5748583
This way, you can access the path to a local function.

Reply:
Implemented, works great.